src/lua/LuaPiGui.cpp

```diff
@@ -2549,6 +2549,71 @@ static int l_pigui_drag_int_4(lua_State *l)
 	return 5;
 }
 
+static int l_pigui_increment_drag(lua_State *l)
```

Comment:
I'd recommend that the actual widget behavior (especially of this length) should go in `src/pigui/Widgets.cpp`, and this file should only be concerned with parameter transport in and out of Lua.

Reply:
Done!